README.md

```diff
@@ -260,36 +177,121 @@ quantized_model = nncf.quantize(model, calibration_dataset)
 
 </details>
 
+
+[//]: # (NNCF provides full  [samples]&#40;#post-training-quantization-samples&#41;, which demonstrate Post-Training Quantization usage for PyTorch, TensorFlow, ONNX, OpenVINO.)
+
+### Training-Time Compression
+
+Training-Time Compression pipeline allows to fine-tune the quantized model in order to improve on the possible accuracy drop introduced by quantization. The basic workflow steps are:
+1) Create an NNCF config object from (a) JSON configuration file and (b) initialization dataloader.
+2) Pass your model together with the config to the `create_compressed_model()` function which returns (a) compression controller and (b) compressed model.
+3) Finetune the compressed model as a regular PyTorch/Tensorflow module.
+4) Export the finetuned model using the compression controller.
+
+Expand the blocks below for PyTorch and Tensorflow usage examples: 
+
+<details><summary><b>PyTorch</b></summary>
+
+```python
+import torch
+import nncf.torch  # Important - must be imported before any other external package that depends on torch
```

Comment:
As found in PR with auto formatting. Isort can automatically move move nncf below other imports.  It can confused users who used isort or .
Suggest to add to readme `# isort: off
```python
# isort: off
# Important - must be imported before any other external package that depends on torch
import nncf.torch  
from nncf import NNCFConfig
from nncf.torch import create_compressed_model, register_default_init_args
# isort: on
```